lib/src/layer/polygon_layer/painter.dart

```diff
@@ -209,11 +223,25 @@ base class _PolygonPainter<R extends Object>
 
         for (final holeOffsets in holeOffsetsList) {
           filledPath.addPolygon(holeOffsets, true);
+
+          /* https://github.com/flutter/flutter/issues/44572          
+          filledPath = Path.combine(
+            PathOperation.difference,
+            filledPath,
+            Path()..addPolygon(holeOffsets, true),
+          );*/
```

Comment:
How about a TODO with information when this can get removed or re-added? I think this would be better than regular commented out code.

Reply:
Yep, I'll do this. The linked issue is of some use, but basically this code is just potentially more efficient than what we're doing - but not sure. Basically, once that issue is closed, we should test again to check the most efficient combination of things. I'll add that to the code.

Reply:
Sounds good! 👍